src/main/webapp/resources/js/pages/projects/list/ProjectsTable.jsx

```diff
@@ -186,35 +187,34 @@ export function ProjectsTable() {
   );
 
   return (
-    <PageWrapper
-      title={getI18N("ProjectsTable_header")}
-      headerExtras={
-        <Row gutter={12} style={{ marginRight: 18 }}>
-          <Col span={18}>
-            <Input.Search onSearch={onSearch} />
-          </Col>
-          <Col span={6}>
-            <Dropdown overlay={exportMenu} key="export">
-              <Button>
-                {getI18N("ProjectsTable_export")} <Icon type="down" />
-              </Button>
-            </Dropdown>
-          </Col>
-        </Row>
-      }
-    >
-      <Table
-        style={{ margin: "6px 24px 0 24px" }}
-        rowKey={record => record.id}
-        loading={loading}
-        pagination={{
-          total: total,
-          pageSize: state.pageSize
-        }}
-        columns={columns}
-        dataSource={projects}
-        onChange={handleTableChange}
-      />
+    <PageWrapper title={getI18N("ProjectsTable_header")}>
+      <>
```

Comment:
These empty brackets aren't needed

Reply:
Removed in c3667bba615bc258475e35681e1f68f6f7a210fb